source/mainnet/smart-contracts/best-practices/development.rst

```diff
@@ -0,0 +1,203 @@
+.. _sc-development-best-practices:
+
+==========================
+Development best practices
+==========================
+
+This document provides guidelines for developing smart contracts.
+It starts with some general thoughts about smart contract development and then gives more details about writing smart contracts in Rust for Concordium.
+
+Mindset
+========
+
+Smart contract development involves many risks that do not show up in, for example, web programming:
+
+- the cost of mistakes is very high;
+- possibilities for fixing bugs are limited;
+- the area is evolving constantly, with new vulnerabilities being discovered regularly;
+- malicious parties deliberately try to break your smart contracts and steal the money.
```

Comment:
I don't know if "steal the money" is too specific. There are types of hacks that don't involve money directly.

Reply:
True, but stealing funds are one of the common goals (and probably the main motivation).
Maybe, "try to steal funds or break your contract in some other way"?

Reply:
Yea, just phrase it as an example of things that can occur.